src/apps/app/app.element.ts

```diff
@@ -1,4 +1,6 @@
 import '@umbraco-ui/uui-css/dist/uui-css.css';
+import '../../shared/css/custom-properties.css';
```

Comment:
Seems like you added this as a tsconfig path alias, so consider using that alias here.